docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,451 @@
+      "text/plain": [
+       "<Figure size 576x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "exp1.analysis.set_options(p0=None, plot=True)\n",
+    "expdata1 = exp1.run(backend=backend, shots=2000)\n",
+    "expdata1.block_for_results()  # Wait for job/analysis to finish.\n",
+    "\n",
+    "# Display the figure\n",
+    "display(expdata1.figure(0))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "DbAnalysisResultV1\n",
+      "- name: @Parameters_T2HahnAnalysis\n",
+      "- value: [4.73150194e-01 5.03648438e-01 1.98283181e-05] ± [5.15456349e-03 3.04084131e-03 5.77525843e-07]\n",
+      "- χ²: 0.7488240853624647\n",
+      "- quality: good\n",
+      "- extra: <4 items>\n",
+      "- device_components: ['Q0']\n",
+      "- verified: False\n",
+      "DbAnalysisResultV1\n",
+      "- name: T2\n",
+      "- value: 1.982831812408823e-05 ± 5.775258431912853e-07 s\n",
+      "- χ²: 0.7488240853624647\n",
+      "- quality: good\n",
+      "- device_components: ['Q0']\n",
+      "- verified: False\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Print results\n",
+    "for result in expdata1.analysis_results():\n",
+    "    print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Providing initial user estimates\n",
+    "The user can provide initial estimates for the parameters to help the analysis process. Because the curve is expected to decay toward $0.5$, the natural choice for parameter $B$ is $0.5$. When there is no $T_2$ error, we would expect that the probability to measure `1` is $100\\%$, so we will guess that A is $0.5$. In this experiment, `t2hahn` is the parameter of interest. Good estimate for it is the value computed in previous experiments on this qubit or a similar value computed for other qubits."
```

Comment:
```suggestion
    "The user can provide initial estimates for the parameters to help the analysis process. Because the curve is expected to decay toward $0.5$, the natural choice for parameter $B$ is $0.5$. When there is no $T_2$ error, we would expect that the probability to measure `1` is $100\\%$, therefore we will guess that A is $0.5$. In this experiment, `t2hahn` is the parameter of interest. Good estimate for it is the value computed in previous experiments on this qubit or a similar value computed for other qubits."
```

Reply:
Changed as suggested.